arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -574,9 +574,6 @@ void DatabaseFeature::recoveryDone() {
       continue;
     }
 
-    // execute the engine-specific callbacks on successful recovery
-    engine->recoveryDone(*vocbase);
```

Comment:
I bet that may break ArangoSearch recovery logic

Reply:
I don't think so. `recoveryDone(vocbase)` was a no-op in all storage engines but MMFiles.

Reply:
you're right, I messed it up with `DatabaseFeature::recoveryDone()` itself :-)